src/network/networkd-fdb.h

```diff
@@ -19,6 +19,8 @@ struct FdbEntry {
         Network *network;
         NetworkConfigSection *section;
 
+        uint64_t vni;
```

Comment:
If the above `sd_netlink_message_append_u32()` is correct, then maybe this should be `uint32_t`, and use `safe_atou32()` in the conf parser.